security/pkg/server/ca/server.go

```diff
@@ -272,14 +272,12 @@ func (s *Server) authenticate(ctx context.Context) *authenticate.Caller {
 	// TODO: apply different authenticators in specific order / according to configuration.
 	for _, authn := range s.authenticators {
 		u, err := authn.Authenticate(ctx)
-		if err != nil {
-			log.Warnf("Authentication failed: %v", err)
```

Comment:
Why is this being removed?  Too spammy?

Reply:
Yes, it's too spammy.

Reply:
I can change this to Debugf.

Reply:
Another possibility is to do a "if (0 == log_calls++ % 100)" guard (sorry for the C) and keep the natural log severity.  If people really should pay attention to the event then making it Debug may not be the right thing to do.

But I'll leave this to you to address in a subsequent PR if you think that's a good idea.  I'll merge this once the non-flakey tests pass.

Reply:
Cool. I understand that. I can address that separately.
Now I concatenate the error messages from all authentiators and output them only if all of them fail.